src/lib/svg_text_utils.js

```diff
@@ -17,6 +17,17 @@ var Lib = require('../lib');
 var xmlnsNamespaces = require('../constants/xmlns_namespaces');
 var stringMappings = require('../constants/string_mappings');
 
+exports.getDOMParser = function() {
+    if(exports.domParser) {
+        return exports.domParser;
```

Comment:
Lets make `domParser` a module-level global instead of an export - it's unsafe to use in general as it might not exist before calling `exports.getDOMParser()`.